src/foam/dashboard/view/GroupedDAOTableWidget.js

```diff
@@ -32,10 +32,13 @@ foam.CLASS({
     async function init() {
       this.SUPER();
       var self = this;
-      result = await self.__subContext__[self.daoKey].where(self.predicate).select(self.GroupBy.create({
-        arg1: self.arg1,
-        arg2: self.arg2
-      }));
+
+      result = await self.__subContext__[self.daoKey].where(self.predicate)
+        .select(self.GROUP_BY(
```

Comment:
Why did we need to change from `self.GroupBy` to `self.GROUP_BY`?

Reply:
Shorter and prepares args better.